### ISSUE TITLE
[Ember Engine] `nav.link-to` does not prefix mount point to the route names

### ISSUE THREAD

**Issue by sumeetattree** [kaliber5/ember-bootstrap#805]
When using `nav.link-to` inside `ember-engines` the routes are not prefixed with the engine mount point and generates incorrect urls. A related issue is referenced here on the `ember-engines` repository https://github.com/ember-engines/ember-engines/issues/304

```hbs
{{nav.link-to 'Invalid Route' 'blog'}}
```
```
// Expected output
'engine-name.blog'

// Actual output
'blog'
```

As mentioned here https://github.com/ember-engines/ember-engines/issues/304 , it is due to `ember-bootstrap` extending `LinkComponent` from `@ember/routing/link-component` instead of `ember-engines/components/link-to`

Output from `ember bootstrap:info`:

```
Npm packages:
ember-bootstrap: ^2.6.0 -> 2.6.1
ember-cli: ~3.8.1 -> 3.8.2
bootstrap: ^4.2.1 -> 4.3.1
bootstrap-sass: n/a
ember-cli-sass: ^7.2.0 -> 7.2.0
ember-cli-less: n/a
Bower packages:
bootstrap: n/a
bootstrap-sass: n/a
ember-bootstrap configuration:
bootstrapVersion: 4
importBootstrapFont: false
importBootstrapCSS: false
```

**Comment by simonihmig** [comment]
#783 will deprecate `nav.link-to`, so I think we won't fix it. The new proposed way is to use the plain `{{link-to}}`. There will however most probably be still an issue with setting the `active` class on the `nav.item` (which was the only reason to have `nav.link-to` in the first place), as it will ask the router service if the given link params match the active route, but without taking that mount point into consideration again. 🤔

I'll keep this open therefore...

In the meantime you could use `{{link-to}}` instead of `nav.link-to` to get correct links, but then you would have to manage the `active` state of the `nav.item` by yourself: https://www.ember-bootstrap.com/api/classes/Components.NavItem.html#property_active

**Comment by sumeetattree** [comment]
> In the meantime you could use {{link-to}} instead of nav.link-to to get correct links, but then you would have to manage the active state of the nav.item by yourself: https://www.ember-bootstrap.com/api/classes/Components.NavItem.html#property_active

Thanks simon. That's what I ended up doing.

**Comment by simonihmig** [comment]
This should be fixed by #861, as there we don't extend form `LinkComponent` anymore, but use the plain `<LinkTo>` directly. Closing this therefore, feel free to reopen should the error be still present with the upcoming v3.0.0 release.
